### ISSUE TITLE
midterm report peer review

### ISSUE THREAD

**Issue by StephenChen1998** [kothiyalp15/ORIE-5741-FIFA#9]
This project is going to study how FIFA player ratings could help predict a soccer player's actual market value. The primary data set is FIFA player Database from previous editions of the game, and they want to use their model to make predictions on FIFA 22.The goal of this object is to help company to make informed decisions to future transfers and know how their investment in new players will pan out financially.

3 things I like about the midterm report:

* Good data exploration as well as data visualization. You apply Tableau to visualize market value across all players v.s. one of four general positions
* Good proposing including log transformation and many-hot encoding. Moreover, you also consider the zero value and complete the feature selection before modeling.
* Good awareness of applying PCA approach to reduce overfitting as well as awareness of splitting dataset into training, test and validation set to get a better prediction model.

3 things I think might get improved:

* For the data cleaning part, I think maybe you could also try to find a better way like one-hot encoding rather than many-hot encoding as we have learnt in lecture, and you could try to use feature engineering approaches like normalization for your dataset to increase the interpretability and effectiveness of the modeling results.
* For the feature selection part, I think maybe you could figure out a method to compare the significance of different variables to the target so that you could confirm your feature selection results.
* For modeling, maybe you could try some other models like random forest or some regularization terms like ridge or lasso to improve your models. I think they might help you enhance the prediction accuracy and also reduce overfitting a bit.

Overall, I think your idea is really innovative and I am looking forward to seeing your analysis results.
